srv/salt/_modules/osd.py

```diff
@@ -76,15 +76,21 @@ def configured(**kwargs):
     """
     osds = []
     devices = []
+    # That doesn't allow mixed configurations
+    # storage[osds] OR storage[data+journals]
```

Comment:
I'd like to leave it here as a marker for a future issue, 
will assign it an issue though